### PR TITLE
Feature/screenshot (front-end only)

### PR DESCRIPTION
**Pull request by michaelbretagne**
- Take a screenshot when the photo icon is clicked.
- Display the screenshot modal when the screenshot image has been dispatched.
- Allow the user to click on social media icons to share their screenshot. (front-end only, for now, the sharing part is not coded).
- Implement success/error modal after a screenshot is shared (front-end only, so now the response is hardcoded to true for Instagram and false for the other social media).
- Modify electric-guitar.svg and electric-guitar-success.svg. It wasn't showing properly with the clip-path element.
- Add social icons (downloaded SVGs from font-awesome)
- Add left-curve-arrow.svg (Designed by Nab)
- Modify the fill color of some SVGs so we can easily change their colors with CSS (darker color when they are active/click).
- Remove if statement in Sketch.js waiting for users to click on the play button to start loading the song into p5. Now the song will start loading the moment we are in App.js.

### CODE REVIEW

File: src/components/ScreenshotModal/BackToPlayer.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const BackToPlayer = ({ hideSocialIcons }) => {
+    const dispatch = useDispatch();
+
+    const backToPlayerHandler = () => {
+        hideSocialIcons();
+        dispatch(clearScreenshot());
+    };
+    return (
+        <div className={classes.backToPlayer} onClick={backToPlayerHandler}>
+            <p>Back to PLayer</p>
```
Comment (ArnasDickus) [inline src/components/ScreenshotModal/BackToPlayer.js]: Back to Player not PLayer

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding PR. So everything looks good. But I see a few problems:

1) A black background is out of place. I suggest adding the same BG as player background: #595757
2) As you already mentioned there is no sharing functionality. Since this PR is already big, you don't need to implement it.

File: src/components/ScreenshotModal/BackToPlayer.js (hunk 1 above)
Comment (michaelbretagne) [reply src/components/ScreenshotModal/BackToPlayer.js]: Good catch ;-)

**Comment by michaelbretagne** [comment]
- Added global variable into /globalScss/variables.scss

- Change ScreenshotModal.js, Error.js and Success.js background color to #595757 ($grey).

- Fix typo in BackToPlayer.js.

**Review by ArnasDickus — APPROVED** [review]

**Pull request merged**